Added my own advice to the file

Alan23bh opened the pull request.

twosavoie commented:
Hi @Alan23bh,

This is a great start! Well done adding your advice file to the advice folder.

Let's take a look at the link to your advice file in the index.html file. Currently, the link goes through your computer's desktop: `<a href="file:///Users/alanmac/Desktop/git-practice/advice/Alan-H-advice.html">Alan Hernandez's advice</a><br>`. Since a server can't access your desktop (fortunately! 🙂), the link is broken. 

Instead, the path should go directly to the `advice` folder to find the `Alan-H-advice.html` file. What can you change about the path so that link works? You can use the other student's link as an example of how the path is written.

If you have any questions, let us know!

Best,
Lisa

Alan23bh commented:
I think I may have fixe the link problem, I should've known! If I am not mistaken I have fixed the issue and added/committed it back to the repo with it resolved. Please let me know If i added it and Pull Requested it incorrectly! 
Thank you so much for your feedback 🙏

twosavoie commented:
Hi @Alan23bh,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).

Best,
Lisa
